src/openslide-vendor-synthetic.c

```diff
@@ -124,14 +124,8 @@ static bool decode_dicom(const void *data, uint32_t len,
     g_prefix_error(err, "Reading metadata: ");
     return false;
   }
-  g_autoptr(DcmBOT) bot = dcm_filehandle_read_bot(&dcm_error, fh, metadata);
-  if (!bot) {
-    _openslide_dicom_propagate_error(err, dcm_error);
-    g_prefix_error(err, "Reading BOT: ");
-    return false;
-  }
   g_autoptr(DcmFrame) frame =
-    dcm_filehandle_read_frame(&dcm_error, fh, metadata, bot, 1);
+    dcm_filehandle_read_frame(&dcm_error, fh, 1);
```

Comment:
We could also drop the `dcm_filehandle_read_metadata` call, right?

Reply:
It's needed to scan the first part of the header (it side-effects the filehandle), which is a little ugly. 

This should probably be fixed in a later libdicom.

Reply:
Fair enough.  Can we add a comment to that effect above the `read_metadata` call so the purpose is clear?

Reply:
Ah, I meant on the call in `openslide-vendor-synthetic.c`.  The one in `openslide-vendor-dicom.c` doesn't need a comment because we're actually using the returned metadata.

Reply:
Ah! I was a little puzzled, OK, done.